### PR TITLE
YOLOX supports fp16 training

### PR DESCRIPTION
**Pull request by hhaAndroid**
## Motivation

YOLOX supports fp16 training

Resolve: https://github.com/open-mmlab/mmdetection/issues/5981

### CODE REVIEW

File: mmdet/models/dense_heads/yolox_head.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# Copyright (c) OpenMMLab. All rights reserved.
```
Comment (RangiLyu) [inline mmdet/models/dense_heads/yolox_head.py:1]: Why delete this?

Comment (hhaAndroid) [reply mmdet/models/dense_heads/yolox_head.py:1]: Oh, Maybe I forgot to sync.

**Review by ZwwWayne — APPROVED** [review]

**Comment by ZwwWayne** [comment]
suggest to leave comments in the pr message to indicate 1 whether it can run the training, 2 whether the performance is verified as expected or we leave the benchmark issue in a future pr

**Comment by hhaAndroid** [comment]
> suggest to leave comments in the pr message to indicate 1 whether it can run the training, 2 whether the performance is verified as expected or we leave the benchmark issue in a future pr

ok

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**